nodejs/eks/cni.ts

```diff
@@ -100,37 +97,20 @@ function computeVpcCniYaml(yamlPath: string, args: VpcCniInputs): string {
         env.push({name: "WARM_IP_TARGET", value: args.warmIpTarget.toString()});
     }
     // Return the computed YAML.
-    return cniYaml.map(o => `---\n${jsyaml.safeDump(o)}`).join("");
+    return daemonSet;
 }
 
 function applyVpcCniYaml(yamlPath: string, args: VpcCniInputs) {
-    // Dump the kubeconfig to a file.
-    const tmpKubeconfig = tmp.fileSync();
-    fs.writeFileSync(tmpKubeconfig.fd, args.kubeconfig);
-
-    // Compute the required CNI YAML and dump it to a file.
-    const tmpYaml = tmp.fileSync();
-    fs.writeFileSync(tmpYaml.fd, computeVpcCniYaml(yamlPath, args));
-
-    // Call kubectl to apply the YAML.
-    childProcess.execSync(`kubectl apply -f ${tmpYaml.name}`, {
-        env: { ...process.env, "KUBECONFIG": tmpKubeconfig.name },
-    });
+    const daemonSet = computeVpcCniYaml(yamlPath, args);
+    const replaced = pulumi.runtime.invoke("kubernetes:kubernetes:kubectlReplace", daemonSet);
```

Comment:
This is an async call - right? But we won't await it finishing as currently written.  I believe we will need to do that or this will allow other work to continue without this being complete (and may even not bubble up errors correctly in case this fails.